test/libsolidity/util/TestFileParserTests.cpp

```diff
@@ -42,11 +43,10 @@ using Mode = FunctionCall::DisplayMode;
 namespace
 {
 
-vector<FunctionCall> parse(string const& _source)
+vector<FunctionCall> parse(string const& _source, std::map<std::string, Builtin> const& _builtins = {})
 {
 	istringstream stream{_source, ios_base::out};
-	TestFileParser parser{stream, {}};
-	return parser.parseFunctionCalls(0);
+	return TestFileParser{stream, _builtins}.parseFunctionCalls(0);
```

Comment:
Interestingly this would have silently fixed the bug from #11416 :)

It's a good thing but I guess it would throw a wrench into @axic's efforts to reproduce it, possibly without making him aware that it's no longer reproducible  :P

Reply:
Lucky it wasn't merged then, as having some ASan issue silently go away is rather annoying.

However as for me reproducing it, somehow need to get past the jsoncpp issue first 😅